.github/workflows/py-tests.yml

```diff
@@ -74,5 +74,7 @@ jobs:
         run: |
           pip install -e src/py
       - name: Run with pytest
+        env:
+          SECRET_KEY: secret_key
```

Comment:
we needed this previously but i removed this since we're not getting the secret_key from the enironment anymore (needed previously to establish test client)